### PR TITLE
chore(ci): build packages on tag builds

### PR DESCRIPTION
**Pull request by geigerzaehler**
We build the packages on tag builds so that they can easily be downloaded.

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -537,10 +537,12 @@ gsutil cp dist/radicle-upstream-0.1.2.dmg gs://releases.radicle.xyz
 To be able to upload packages to the GCS bucket you will need the appropriate
 permissions. Reach out to a co-worker if you don’t have them.
 
-You should also copy the Linux `.AppImage` and `.snap` packages to the same
-location. You can get them from the [CI build][ar] on Buildkite. Then run
+After this you also need to obtain and upload the Linux packages.
 
 ```bash
+curl -sSLO https://builds.radicle.xyz/radicle-upstream/v0.1.2/radicle-upstream-0.1.2.AppImage
+curl -sSLO https://builds.radicle.xyz/radicle-upstream/v0.1.2/radicle-upstream-0.1.2.snap
```
Comment (rudolfs) [inline DEVELOPMENT.md:544]:
```suggestion
```

Comment (rudolfs) [reply DEVELOPMENT.md:544]: No `.snap` for now.

**Review by xla — APPROVED** [review]
💰 🌄 ⏺ ⚠️

**Review by rudolfs — APPROVED** [review]

**Comment by geigerzaehler** [comment]
Merged in https://github.com/radicle-dev/radicle-upstream/commit/99852e32f2fa7956108bebc9ca7af8fd643f2771

**Pull request closed without merging**